app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/cumulativeTrendControls.jsx

```diff
@@ -83,6 +85,10 @@ const validators = {
 
 @injectIntl
 @connect((state) => ({
+  levelsAttributes: formValueSelector(WIDGET_WIZARD_FORM)(
```

Comment:
Please move selector creation out of this function (above the class definition) like this:
`const valuesSelector = formValueSelector(WIDGET_WIZARD_FORM);`